masonite/middlewares/VerifyCsrfToken.py

```diff
@@ -0,0 +1,33 @@
+from masonite.exceptions import InvalidCSRFToken
+
+
+class VerifyCsrfToken(object):
+    """
+    Verify csrfk token middleware
+    """
+
+    exempt = []
+
+    def __init__(self, Request):
```

Comment:
When you add the `CSRFProvider` then you'll need to also but `CSRF` in this constructor